lib/testing/src/lib/core/actions/api.service.ts

```diff
@@ -86,6 +86,15 @@ export class ApiService {
         await this.apiService.login(username, password);
     }
 
+    /**
+     * Login using one of the account profiles from the `browser.params.testConfig`.
+     * Example: loginWithProfile('admin')
+     */
+    async loginWithProfile(profileName: string): Promise<void> {
+        const profile = browser.params.testConfig[profileName];
```

Comment:
shouldn't check if the profile is undefined?

Reply:
added throwing an error, thanks